src/Persistent_cohomology/example/rips_multifield_persistence.cpp

```diff
@@ -104,7 +104,7 @@ void program_options(int argc, char * argv[]
       ("min-field-charac,p", po::value<int>(&min_p)->default_value(2),
        "Minimal characteristic p of the coefficient field Z/pZ.")
       ("max-field-charac,q", po::value<int>(&max_p)->default_value(1223),
-       "Minimial characteristic q of the coefficient field Z/pZ.")
+       "Minimal characteristic q of the coefficient field Z/pZ.")
```

Comment:
```suggestion
       "Maximal characteristic q of the coefficient field Z/pZ.")
```
(the mix of p and q also looks strange, but that's a different problem)